pkg/agent/multicast/mcast_controller.go

```diff
@@ -432,7 +446,11 @@ func (c *Controller) syncGroup(groupKey string) error {
 	}
 	status := obj.(*GroupMemberStatus)
 	memberPorts := make([]uint32, 0, len(status.localMembers)+1)
-	memberPorts = append(memberPorts, config.HostGatewayOFPort)
+	if c.flexibleIPAMEnabled {
+		memberPorts = append(memberPorts, config.UplinkOFPort, c.nodeConfig.HostInterfaceOFPort)
```

Comment:
Since we always have `HostGatewayOFPort` in both modes, I wonder if they are the same to stick to that interface for forwarding traffic to the Node itself, and only append `UplinkOFPort` when flexibleIPAM is enabled.
So do the flows.